src/lib/util/AntiUserGateway.ts

```diff
@@ -0,0 +1,225 @@
+// TODO: Switch to @skyra/anti-user-gateway once we move to @klasa/core
+
+import { IdKeyed, RequestHandler } from '@klasa/request-handler';
+import { Gateway, GatewayStorage, Settings, SettingsExistenceStatus, SettingsFolderJson, SettingsUpdateResults } from '@klasa/settings-gateway';
+import { SkyraClient } from '@lib/SkyraClient';
+import { KlasaUserStore } from 'klasa';
+import { container } from 'tsyringe';
+
+/* eslint-disable @typescript-eslint/class-literal-property-style */
+
+// Step 1: Remove UserStore#fetch
+Reflect.deleteProperty(KlasaUserStore.prototype, 'fetch');
+
+// Step 2: Create UserSettings, which is an empty Settings-like class, implements:
+//         Settings, SettingsFolder, and Map<string, unknown>.
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
```

Comment:
Add a short description as to why there is a ts-ignore then eslint won't complain either.